getting_started_remote_sensing_data.md

```diff
@@ -0,0 +1,33 @@
+# Getting Started with Remote Sensing Data
+
+In this article, we will use remote sensing (here on refered to as RS) data to conduct a simple analysis. Out of many satellites that orbit earth collecting RS data, here we will use the data from [LANDSAT](https://landsat.gsfc.nasa.gov/landsat-8/) satellite which is freely available and can be downloaded from [Earth Explorer](https://earthexplorer.usgs.gov). 
+
+## Downloading data
+We will download the data for Bangalore, India. As shown in the screenshot below, we search for the area of interest which in our case is Bangalore. Once you select from the given suggestions, latitude and ongitudes details will be filled in automatiallyand you can select date range for the time you want to download the data (Figure2). We download images for November 1999 and October 2018. Next we need to click on Datasets to select which satellite data we want to download, for us it is LANDSAT 7. While downloading the data, we ensure that the coud cover is less than 10% by adding criteria in Additional Criteria tab. By clicking on footprint (little icon of footnear image thumbnail), we can see the footprint of the scene (the geographic area covered by the image) on the map as shown in Figure 3. Once you have selected the data, we order it (Since the data is freely available you won't be charged for it :)). We might have to wait a little while before data can be downloaded (You will receive an email from USGS when the data is ready for download). 
```

Comment:
```suggestion
We will download the data for Bangalore, India. As shown in the screenshot below, we search for the area of interest which in our case is Bangalore. Once you select from the given suggestions, latitude and longitude details will be filled in automatically and you can select date range for the time you want to download the data (Figure 2). We download images for November 1999 and October 2018. Next we need to click on Datasets to select which satellite data we want to download, for us it is LANDSAT 7. While downloading the data, we ensure that the cloud cover is less than 10% by adding criteria in Additional Criteria tab. By clicking on footprint (little icon of foot near the image thumbnail), we can see the footprint of the scene (the geographic area covered by the image) on the map as shown in Figure 3. Once you have selected the data, we order it (Since the data is freely available you won't be charged for it :)). We might have to wait a little while before data can be downloaded (You will receive an email from USGS when the data is ready for download). 
```